doc-surrealdb_versioned_docs/version-1.x/surrealql/functions/search.mdx

```diff
@@ -63,7 +63,7 @@ SELECT id, title, search::score(1) AS score FROM book
 The `search::highlight` highlights the matching keywords for the predicate reference number.
 
 ```surql title="API DEFINITION"
-search::highlight(string, string, number) -> string | string[]
+search::highlight(string, string, number, [boolean]) -> string | string[]
```

Comment:
Interesting, could this be our first function with an optional parameter? I was looking through the rest and don't see an example of the syntax but the [] part looks correct. Though I think `boolean` in the signature should be changed to `bool`, e.g. here is one function using the keyword:

```
parse::url::fragment(string) -> bool
```

Reply:
I think boolean is more descriptive since we have https://surrealdb.com/docs/surrealdb/surrealql/datamodel/booleans